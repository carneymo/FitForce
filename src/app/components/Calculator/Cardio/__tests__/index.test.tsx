import * as React from 'react';
import { render } from '@testing-library/react';

import { Cardio } from '..';

describe('<Cardio  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Cardio age={''} gender={''} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
