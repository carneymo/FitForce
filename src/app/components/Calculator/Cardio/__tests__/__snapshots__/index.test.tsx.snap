// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cardio  /> should match snapshot 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthLg css-1o6ysxg-MuiContainer-root"
>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 MuiCard-root css-sc5siq-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-2 css-1o7apob-MuiGrid-root"
      >
        <div
          class="MuiToggleButtonGroup-root MuiToggleButtonGroup-vertical css-1g71zaa-MuiToggleButtonGroup-root"
          role="group"
        >
          <button
            aria-label="run"
            aria-pressed="true"
            class="MuiButtonBase-root MuiToggleButton-root Mui-selected MuiToggleButton-sizeMedium MuiToggleButton-standard MuiToggleButtonGroup-grouped MuiToggleButtonGroup-groupedVertical css-ueukts-MuiButtonBase-root-MuiToggleButton-root"
            tabindex="0"
            type="button"
            value="run"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="DirectionsRunIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M13.49 5.48c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm-3.6 13.9 1-4.4 2.1 2v6h2v-7.5l-2.1-2 .6-3c1.3 1.5 3.3 2.5 5.5 2.5v-2c-1.9 0-3.5-1-4.3-2.4l-1-1.6c-.4-.6-1-1-1.7-1-.3 0-.5.1-.8.1l-5.2 2.2v4.7h2v-3.4l1.8-.7-1.6 8.1-4.9-1-.4 2 7 1.4z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            aria-label="hamr"
            aria-pressed="false"
            class="MuiButtonBase-root MuiToggleButton-root MuiToggleButton-sizeMedium MuiToggleButton-standard MuiToggleButtonGroup-grouped MuiToggleButtonGroup-groupedVertical css-ueukts-MuiButtonBase-root-MuiToggleButton-root"
            tabindex="0"
            type="button"
            value="hamr"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="TransferWithinAStationIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.49 15.5v-1.75L14 16.25l2.49 2.5V17H22v-1.5zm3.02 4.25H14v1.5h5.51V23L22 20.5 19.51 18zM9.5 5.5c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zM5.75 8.9 3 23h2.1l1.75-8L9 17v6h2v-7.55L8.95 13.4l.6-3C10.85 12 12.8 13 15 13v-2c-1.85 0-3.45-1-4.35-2.45l-.95-1.6C9.35 6.35 8.7 6 8 6c-.25 0-.5.05-.75.15L2 8.3V13h2V9.65l1.75-.75"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-sm-10 css-17p3wh3-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-fgvrm7"
        >
          <strong>
            1.5 Mile Run
          </strong>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 css-19gy2ei-MuiGrid-root"
            >
              <div
                class="MuiFormControl-root MuiFormControl-marginDense MuiTextField-root css-13e98gt-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1jy569b-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="true"
                  for="milerun-minutes"
                  id="milerun-minutes-label"
                >
                  mm
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
                    id="milerun-minutes"
                    type="text"
                    value="00"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="css-14lo706"
                    >
                      <span>
                        mm
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiFormControl-marginDense MuiTextField-root css-xnm3t8-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1jy569b-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="true"
                  for="milerun-seconds"
                  id="milerun-seconds-label"
                >
                  ss
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
                    id="milerun-seconds"
                    type="text"
                    value="00"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="css-14lo706"
                    >
                      <span>
                        ss
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 css-fjbua9-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-e93pxh-MuiTypography-root"
              >
                5.0
              </p>
               
              <p
                class="MuiTypography-root MuiTypography-body1 css-vappet-MuiTypography-root"
              >
                / 20
              </p>
              <div
                class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorError MuiChip-filledError css-lwe8th-MuiChip-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-icon MuiChip-iconSmall MuiChip-iconColorError css-i4bv87-MuiSvgIcon-root"
                  data-testid="WarningAmberIcon"
                  focusable="false"
                  style="color: white;"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 5.99 19.53 19H4.47L12 5.99M12 2 1 21h22L12 2z"
                  />
                  <path
                    d="M13 16h-2v2h2zm0-6h-2v5h2z"
                  />
                </svg>
                <span
                  class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
                >
                  Unsatisfactory
                </span>
              </div>
              <div
                class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorPrimary MuiChip-filledPrimary css-174hymm-MuiChip-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-icon MuiChip-iconSmall MuiChip-iconColorPrimary css-i4bv87-MuiSvgIcon-root"
                  data-testid="RecommendIcon"
                  focusable="false"
                  style="color: white;"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm6 9.8a.9.9 0 0 1-.1.5l-2.1 4.9a1.34 1.34 0 0 1-1.3.8H9a2 2 0 0 1-2-2v-5a1.28 1.28 0 0 1 .4-1L12 5l.69.69a1.08 1.08 0 0 1 .3.7v.2L12.41 10H17a1 1 0 0 1 1 1z"
                  />
                </svg>
                <span
                  class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
                >
                  Satisfactory
                </span>
              </div>
              <div
                class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorSuccess MuiChip-filledSuccess css-li422f-MuiChip-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-icon MuiChip-iconSmall MuiChip-iconColorSuccess css-i4bv87-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  style="color: gold;"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
                <span
                  class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
                >
                  Excellent
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
